pkg/storaged/content-views.jsx

```diff
@@ -454,9 +454,9 @@ function block_description(client, block) {
         if (config) {
             type = C_("storage-id-desc", "Filesystem (encrypted)");
             used_for = mount_point;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230425-100703-453867ac-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)